projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-assign/admin-badges-assign.component.html

```diff
@@ -0,0 +1,46 @@
+<nb-card class="admin-bagdes-assign">
+    <nb-card-header class="d-flex justify-content-space-between align-items-center">
+        Badges
+        <button [routerLink]="['/sys-admin', 'badges']" ghost nbButton shape="round" size="small">Show All Badges</button>
+    </nb-card-header>
+    <nb-card-body>
+        <nb-list>
+            <nb-list-item>
+                <form (submit)="assignBadge()" [formGroup]="assignBadgeForm" class="form-fields d-flex justify-content-space-between">
+                    <div class="input-fields">
+                        <input formControlName="username" nbInput type="text" placeholder="Username"/>
+                        <input formControlName="badge_id" nbInput type="number" placeholder="Badge Id"/>
+                    </div>
+                    <button [disabled]="!assignBadgeForm.valid" nbButton size="medium" status="primary" type="submit">Assign</button>
+                </form>
+            </nb-list-item>
+            <nb-list-item>
+                <nb-list>
```

Comment:
Let's format this nb-list to remove the border of first and last elements

Reply:
done